### PR TITLE
CLOUDSTACK-8723: Verify API call "listUsageRecords" returns usage of new volume created after restore VM

### PR DESCRIPTION
**Pull request by pritisarap12**
After restoring a running VM current ROOT disk gets destroyed and new ROOT disk gets created.
This testcase is to check if volume usage of this newly created volume is listed in listUsageRecords API.

### CODE REVIEW

File: test/integration/testpaths/testpath_usage.py [hunk 1]
```diff
@@ -2879,6 +2882,60 @@ def test_07_positive_tests_usage(self):
...
+            serviceofferingid=self.service_offering.id,
+            zoneid=self.zone.id,
+        )
+
+        volumes_root_list = list_volumes(
+            self.apiclient,
+            virtualmachineid=vm_cluster.id,
+            type='ROOT',
+            listall=True
+        )
```
Comment (nitt10prashant) [inline test/integration/testpaths/testpath_usage.py]: use validate list

File: test/integration/testpaths/testpath_usage.py [hunk 2]
```diff
@@ -2879,6 +2882,60 @@ def test_07_positive_tests_usage(self):
...
+
+        volumeCheck = "Volume Id: " + str(qresultset[0][0]) + " usage time"
+
+        response = self.listUsageRecords(usagetype=6)
+        self.assertEqual(response[0], PASS, response[1])
+        UsageRecords = [record for record in response[1]
+                        if volumeCheck in record.description]
+        # Step 3
+        self.assertNotEqual(
```
Comment (nitt10prashant) [inline test/integration/testpaths/testpath_usage.py]: even  if  UsageRecords is empty test case will pass , is that we are expecting ?

Comment (pritisarap12) [inline test/integration/testpaths/testpath_usage.py]: No, In step 3 we are verifying that UsageRecords should not be None.
or I have updated test case to fail it if UsageRecords  is empty.

**Comment by bhaisaab** [comment]
LGTM

**Pull request merged**